backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/fresh_Attributes.sql

```diff
@@ -19,6 +19,7 @@ SELECT att.ai_value,
 FROM (
          SELECT CASE
                     WHEN a.Value = '1000015' AND av.value = '01'                                          THEN NULL -- ADR & Keine/Leer
+                    WHEN a.Value = '1000015' AND (av.value IS NOT NULL AND av.value != '') THEN coalesce( nullif(trim(printvalue_override), ''), av.name) -- ADR
```

Comment:
what's the point of checking `a.Value = '1000015'`?

I thought because we have IsPrintInDocuments and PrintValueOverride, 
here we don't have to hardcode the 1000015 case anymore!